test/integration-test/src/tests/load.rs

```diff
@@ -25,11 +32,20 @@ fn long_name() {
         .try_into()
         .unwrap();
     name_prog.load().unwrap();
-    name_prog.attach("lo", XdpFlags::default()).unwrap();
+
+    // Create veth interface pair
+    create_veth_pair(TEST_VETH_NAME).unwrap();
+
+    name_prog
+        .attach(TEST_VETH_NAME, XdpFlags::default())
+        .unwrap();
 
     // We used to be able to assert with bpftool that the program name was short.
     // It seem though that it now uses the name from the ELF symbol table instead.
     // Therefore, as long as we were able to load the program, this is good enough.
+
+    // Delete veth interface pait
```

Comment:
```suggestion
    // Delete veth interface pair.
```
You didn't fix it after my previous comment :smiling_face_with_tear: